ledger/daml-on-sql/src/main/scala/on/sql/Cli.scala

```diff
@@ -25,6 +27,13 @@ private[sql] final class Cli(override val defaultConfig: SandboxConfig = Default
       .text("Allows development versions of DAML-LF language and transaction format.")
       .hidden()
 
+    parser
+      .opt[String]("sql-backend-jdbcurl-env")
+      .optional()
+      .text(
+        s"The environment variable containing JDBC connection URL to a Postgres database, including username and password. If present, $Name will use the database to persist its data.")
+      .action((env, config) => config.copy(jdbcUrl = getEnv(env).orElse(config.jdbcUrl)))
```

Comment:
If I specify `--sql-backend-jdbcurl-env=FOO --sql-backend-jdbcurl=BAR`, which one wins?

Reply:
The last one, it is the disadvantage of supporting the option here; I have a pending conversation to see if this is acceptable.

Reply:
I think that's probably acceptable from my point of view. It's the same as providing a switch twice, which is in line with most Unix tools.

Reply:
Yes, that was my thinking as well.

Reply:
@gerolf-da, if it's fine with you I'd merge.